packages/field/src/FieldWrapper.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import { BaseExtensionSDK, Entry, FieldAPI } from '@contentful/field-editor-shared';
+import { HelpText } from '@contentful/forma-36-react-components';
+import { ValidationErrors } from '@contentful/field-editor-validation-errors';
+
+type FieldWrapperProps = {
+  sdk: BaseExtensionSDK;
+  children: React.ReactNode;
+  name: string;
+  required: boolean;
+  field: FieldAPI;
+  className?: string;
+  getEntryURL: (entry: Entry) => string;
+};
+
+export const FieldWrapper: React.FC<FieldWrapperProps> = function ({
+  sdk,
+  children,
+  name,
+  required,
+  field,
+  className,
+  getEntryURL,
+}: FieldWrapperProps) {
+  return (
+    <div className={className}>
+      <HelpText>
+        {name}
+        {required ? ' (required)' : ''}
+      </HelpText>
+
+      {children}
+
+      <ValidationErrors
+        field={field}
+        space={sdk.space}
+        locales={sdk.locales}
+        getEntryURL={getEntryURL}
+      />
```

Comment:
`FieldWrapper` should be also responsible for rendering the help text which comes from `sdk.parameters.instance.helpText`.

In general, before publishing this package we need to make sure that we can use it as is in the main web application and that it has no regressions compared to what we currently have there.

Reply:
@suevalov Help text might also come from `editorInterface.controls[fieldId].settings.helpText`, right?

Reply:
`editorInterface.controls[fieldId].settings` is actually  from where`sdk.parameters.instance` get their values